src/app/(sidebar)/transaction-dashboard/page.tsx

```diff
@@ -67,9 +71,43 @@ export default function TransactionDashboard() {
   const isDataLoaded = Boolean(txDetails);
 
   const { isSorobanTx } = getTxData(txDetails || null);
+  const isTxNotFound = txDetails?.status === "NOT_FOUND";
 
   const queryClient = useQueryClient();
 
+  const EXTERNAL_EXPLORERS = [
+    {
+      id: "stellar-expert",
+      label: "Stellar.Expert",
+      onClick: () => {
+        if (transactionHashInput && network.id) {
+          const seNetwork = getStellarExpertNetwork(network.id);
+          openUrl(`${STELLAR_EXPERT}/${seNetwork}/tx/${transactionHashInput}`);
+        }
+      },
+    },
+    {
+      id: "lumenscan",
+      label: "Lumenscan",
+      onClick: () => {
+        if (transactionHashInput && network.id) {
+          const lsUrl =
+            network.id === "mainnet"
+              ? "https://lumenscan.io/txns"
+              : "https://testnet.lumenscan.io/txns";
+          openUrl(`${lsUrl}/${transactionHashInput}`);
+        }
+      },
+    },
+    {
+      id: "goldsky",
+      label: "Goldsky",
+      onClick: () => {
+        openUrl("https://goldsky.com/chains/stellar");
```

Comment:
why do we link to a general goldsky site without a hash link?

Reply:
Hey @johncanneto, is there an endpoint we could use to retrieve transactions from Goldsky? Thank you in advance 🙏